Source/WebKit/WebProcess/WebPage/Cocoa/UnifiedTextReplacementController.mm

```diff
@@ -104,6 +105,30 @@ static void replaceContentsInRange(WebCore::Document& document, const WebCore::S
     return std::nullopt;
 }
 
+static std::optional<std::tuple<WebCore::Node&, WebCore::DocumentMarker&>> findReplacementMarkerContainingRange(WebCore::Document& document, const WebCore::SimpleRange& range)
+{
+    RefPtr<WebCore::Node> targetNode;
+    WeakPtr<WebCore::DocumentMarker> targetMarker;
+
+    document.markers().forEachOfTypes({ WebCore::DocumentMarker::Type::UnifiedTextReplacement }, [&range, &targetNode, &targetMarker] (auto& node, auto& marker) mutable {
+        auto data = std::get<WebCore::DocumentMarker::UnifiedTextReplacementData>(marker.data());
+
+        auto markerRange = WebCore::makeSimpleRange(node, marker);
+        if (!WebCore::containsForTesting(WebCore::TreeType::Tree, markerRange, range))
```

Comment:
Don't think we should use `containsForTesting`.

Reply:
ooppssss forgot to actually fix this